rust/azure_iot_operations_services/src/azure_device_registry/models/asset.rs

```diff
@@ -226,29 +226,29 @@ pub struct EventStreamDestination {
 //     Storage {path: String},
 // }
 
-/// A reference to the Device and Endpoint within the device
+/// A reference to the Device and Endpoint within the device (connection information) used by brokers to connect that provides data points for this asset.
 #[derive(Clone, Debug)]
 pub struct DeviceRef {
-    /// The name of the device
+    /// Name of the device resource.
     pub device_name: String,
-    /// The endpoint name of the device
+    /// The name of endpoint to use.
```

Comment:
Feel like we've lost some information about the endpoint here

Reply:
don't necessarily disagree, but given the possible subtle nuances of differing language here, I'd rather match the DTDL exactly for this one.